src/python/zquantum/core/measurement.py

```diff
@@ -659,7 +674,11 @@ def concatenate_expectation_values(
         The combined expectation values.
     """
 
-    combined_expectation_values = ExpectationValues(np.zeros(0,))
+    combined_expectation_values = ExpectationValues(
```

Comment:
Actually, it'll be cleaner if you just remove `,` after 0 and blackify it :)

Reply:
Done in d8682df...2c47b89